tests/adapters/persistance/test_recipe_repo_in_memory.py

```diff
@@ -0,0 +1,81 @@
+from cookboot.application.recipe_repo import RecipeRepo
+from cookboot.adapters.persistance.recipe_repo_in_memory import RecipeRepoInMemory
+from cookboot.domain.recipe import Ingridient
+from cookboot.domain.user import UserId
+from typing import Tuple, no_type_check
+import pytest
+
+
+@pytest.fixture
+def ingridients() -> Tuple[Ingridient, ...]:
+    return (Ingridient("avocado"), Ingridient("love"))
+
+
+@pytest.fixture
+def user_id() -> UserId:
+    return "user_007"
+
+
+@pytest.fixture
+def name() -> str:
+    return "yummy avocado"
+
+
+@pytest.fixture
+def repo() -> RecipeRepo:
+    return RecipeRepoInMemory()
+
+
```

Comment:
A test is missing: check if an instance of a RecipeRepoInMemory class can be assigned to a variable of type RecipeRepo.